### ISSUE TITLE
Add Option to Expand or Generate a New Train Set ?

### ISSUE THREAD

**Issue by leomignot** [emilienschultz/activetigger#116]
Not sure, but would it be useful to add the ability to reinject additional texts after initial test dataset creation to increase the size of the training set ? or to generate a completely new training set?

**Comment by emilienschultz** [comment]
No, this is not ok for the general pipeline (the test set is randomly built for this reason at the begining). Maybe drop the current testset & upload a new one ? To discuss with @eollion and @jboelaert

**Comment by emilienschultz** [comment]
I added a feature to increase the trainset with random N empty element from the raw data ; for the moment I won't go further, maybe in the future I will reopen this issue if I change the data structure. It is up to the user to create suitable projet...
